code/lib/cli/src/js-package-manager/JsPackageManager.ts

```diff
@@ -79,7 +80,7 @@ export abstract class JsPackageManager {
       this.runInstall();
     } catch (e) {
       done('An error occurred while installing dependencies.');
-      process.exit(1);
+      throw new HandledError(e);
```

Comment:
I'm not sure if there are implications for the migration scripts in throwing rather than exiting inside these functions. @yannbf do you?

It seems weird to call `process.exit()` however.

Reply:
I believe that's fine! The error will be surfaced in the automigrations like:
`❌ error when running XXX migration`